libecole/include/ecole/reward/solvingtime.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "ecole/reward/abstract.hpp"
+#include "ecole/scip/type.hpp"
+
+namespace ecole::reward {
+
+class SolvingTime : public RewardFunction {
+public:
+	SolvingTime(bool wall_ = false) : wall(wall_) {}
+	void reset(scip::Model& model) override;
+	Reward obtain_reward(scip::Model& model, bool done = false) override;
+
+private:
+	bool const wall = false;
```

Comment:
While using `const` is a good practice in C++, it should be avoided for class attribute.
It has undesired effects such as disabling the assignments.

Reply:
Done.